CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to rust-GameDig
+This project is very open to new suggestions, additions and/or changes, these 
+can come in the form of *discussions* about the project's state, *proposing a 
+new feature*, *holding a few points on why we shall do X breaking change* or
+*submitting a fix*.
+
+## Communications
+GitHub is the place we use to track bugs and discuss new features/changes,
+although we have a [Discord](https://discord.gg/NVCMn3tnxH) server for the
+community, all bugs, suggestions and changes will be reported on GitHub 
+alongside with their backing points to ensure the transparency of the project's
+development.
+
+## Issues
+Before opening an issue, check if there is an existing relevant issue first, 
+someone might just have had your issue already, or you might find something 
+related that could be of help.
+
+When opening a new issue, make sure to fill the issue template. They are made
+to make the subject to be as understandable as possible, not doing so may result 
+in your issue not being managed right away, if you don't understand something
+(be it regarding your own problem/the issue template/the library), please state 
+so.
+
+## Development
+Note before contributing that everything done here is under the [MIT](https://opensource.org/license/mit/) license.
+### Naming
+Naming is an important matter, and it shouldn't be changed unless necessary.
+
+Game **names** should be added as they appear on steam (or other storefront 
+if not listed there) with the release year appended in brackets (except when the 
+release year is already part of the name).  
+If there is a mod that needs to be added (or it adds the support for server 
+queries for the game), its name should be composed of the game name, a separating
+**bracket**, the mod name and the release year as specified previously
+(e.g. `Grand Theft Auto V - FiveM (2013)`).
+
+A game's **identification** is lowercased only and be created following these rules:
```

Comment:
We also need to specify that it is lowercase alphanumeric (symbols are stripped).